### PR TITLE
Discovery UI

### PR DESCRIPTION
**Pull request by nrleuenb**
Revamped carousel cards, header adjustments, lovely (stock) banner, pretty color scheme! 

One hacky area: renaming the tags. Didn't want to mess with the tags database so close to the deadline, especially since I'm not familiar with it.

### CODE REVIEW

File: app/assets/stylesheets/header.scss [hunk 1]
```diff
@@ -59,4 +78,8 @@
   cursor:pointer;
   overflow: hidden;
   outline:none;
-}
\ No newline at end of file
+}
+
+.admin {
+  color: $selected-color;
+}
```
Comment (ErikOstien) [inline app/assets/stylesheets/header.scss:85]: Stylesheets look good, and updated navbar css looks good

File: app/views/application/_header.html.haml [hunk 2]
```diff
@@ -29,4 +29,4 @@
             %i.fa.fa-plus
           .dropdown-content
             = button_to 'Add Product', new_admin_product_path, {method: :get, class: 'prod-link'}
-            = button_to 'Add Vendor', new_admin_vendor_path, {method: :get, class: 'prod-link'}
\ No newline at end of file
+            = button_to 'Add Vendor', new_admin_vendor_path, {method: :get, class: 'prod-link'}
```
Comment (ErikOstien) [inline app/views/application/_header.html.haml:32]: header view looks good. No major changes

File: app/views/discovery/_carousel.html.haml [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
+      - if list
+        - list[0..14].each do |vendor|
+          - if vendor.picture.present?
+            = render 'carousel_element', item: vendor, item_path: vendor_path(vendor.id), tag_snaked: tag_snaked
 
+    - if type == :product
+      - if list
+        - list[0..14].each do |product|
+          - if product.picture.present?
+            = render 'carousel_element', item: product, item_path: product_path(product.id), tag_snaked: tag_snaked
```
Comment (ErikOstien) [inline app/views/discovery/_carousel.html.haml:21]: Updated carousel view looks good

**Review by ErikOstien — APPROVED** [review]
This PR updates the discovery page. The CSS files look good and the views properly implement them. The updated carousel code looks good and the website seems to work well on all platforms, including iPhone, iPad, Laptop.

File: app/assets/stylesheets/header.scss [hunk 4]
```diff
@@ -5,6 +5,25 @@
...
+    justify-content: center;
+    float: none;
+    padding-bottom: 1vh;
+    width: 100vw;
+  }
+}
+
 // Dropdown for adding a new product or vendor
 .dropdown-content form:hover {
   background-color: $gray-color;
```
Comment (jlabowitz) [inline app/assets/stylesheets/header.scss:29]: The color of me hovering over an item in the dropdown is almost black. You can barely make out the text.

Comment (nrleuenb) [reply app/assets/stylesheets/header.scss:29]: You're on the wrong branch -- try switching to `discover-ui`; I think you're on the old corrupted reference branch `discoverY-ui`

**Review by jlabowitz — APPROVED** [review]

**Pull request merged**